python/docs/Makefile

```diff
@@ -1,19 +1,44 @@
 # Makefile for Sphinx documentation
 #
 
+ifndef SPHINXBUILD
+ifndef SPHINXPYTHON
+SPHINXBUILD = sphinx-build
+endif
+endif
+
+ifdef SPHINXBUILD
```

Comment:
Is there a way to check that Sphinx build is running with Py3 as done bellow?

Reply:
I couldn't find an easy way.